### ISSUE TITLE
clipboard-access-watcher would be a better name for this project

### ISSUE THREAD

**Issue by kanihal** [dethos/clipboard-watcher#1]
Usually, a clipboard-watcher like ClipQ watches for changes in clipboard content.
This project implements the functionality of watching **accesses** to clipboard, hence clipboard-access-watcher would be a better name.

**Comment by dethos** [comment]
Yes, you are right. The current name isn't a great fit, it's just the name that quickly came to my mind when I started the experiment. Not sure if renaming it now will be a good decision, since it will break any existing links to it.
